models/git/protected_branch_list.go

```diff
@@ -33,7 +33,7 @@ func (rules ProtectedBranchRules) sort() {
 				return true
 			}
 		} else if rules[j].isPlainName {
-			return true
+			return false
 		}
```

Comment:
It could be clear by:
```diff
-		if rules[i].isPlainName {
-			if !rules[j].isPlainName {
-				return true
-			}
-		} else if rules[j].isPlainName {
-			return false
-		}
+		if rules[i].isPlainName != rules[j].isPlainName {
+			return rules[i].isPlainName
+		}

```